CanvasColorSpaceProposal.md

```diff
@@ -1,52 +1,29 @@
 # Color managing canvas contents
 
 ## Use Case Description
-* Contents displayed through a canvas element should be color managed in order to minimize differences in appearance across browsers and display devices. Improving color fidelity matters a lot for artistic uses (e.g. photo and paint apps) and for e-commerce (product presentation).
```

Comment:
The mention of photo / paint / e-commerce apps seems like good motivation for the use cases to me.

Reply:
Re-added.

Reply:
Thanks.